typescript/infra/scripts/middleware/circle-relayer.ts

```diff
@@ -28,10 +26,11 @@ async function check() {
     'middleware/liquidity-layer',
   );
   const addresses = readJSON(dir, 'addresses.json');
-  const contracts = buildContracts(
+  const { contracts } = HyperlaneApp.buildContracts(
```

Comment:
why is this on App now?

Reply:
I guess it felt reasonable since app is where we collect addresses into an object